app/models/katello/glue/elastic_search/content_view_puppet_environment.rb

```diff
@@ -5,26 +5,6 @@ module Glue::ElasticSearch::ContentViewPuppetEnvironment
     def self.included(base)
       base.send :include, Ext::IndexedModel
```

Comment:
going through this now, I get undefined method `disable_auto_reindex!' for #<Katello::ContentViewPuppetEnvironment:0x000000153c0ec8> when testing ../katello/spec/controllers/content_search_controller_spec.rb if I remove line 6, since that method is in Ext::IndexedModel.  any ideas of a workaround?